xonsh/history.py

```diff
@@ -432,26 +621,30 @@ def _gc(ns, hist):
 
 
 _MAIN_ACTIONS = {
-    'show': _show,
+    'session': History.show,
```

Comment:
Hi @JohnLunzer - I guess all that I want is for there to be a show command that displays only this session and for that to be the default behaviour.  Lots of CLI have redundant commands/aliases, git checkout vs git co for example, so I don't really see this being a big deal.  I think if you called your session 'show' from the cli, I would be happy. I am not saying you need to add the old function back in, just preserve the interface.